audformat/core/utils.py

```diff
@@ -212,6 +212,58 @@ def concat(
         return df
 
 
+def duration(
+        obj: typing.Union[pd.Index, pd.Series, pd.DataFrame],
+        *,
+        num_workers: int = 1,
+) -> pd.Timedelta:
+    r"""Duration of all entries present in the object.
```

Comment:
Maybe change to "Total duration" or "Summed duration" to make it more obvious that the duration of the single entries is summed up?

Reply:
I changed to "Total duration"